backend/module/eCampCore/src/EntityService/PeriodService.php

```diff
@@ -126,6 +127,34 @@ protected function deleteEntity(BaseEntity $entity) {
         return $period;
     }
 
+    /**
+     * @param $entity
+     */
+    protected function validateEntity(BaseEntity $entity) {
+        /** @var Period $period */
+        $period = $entity;
+
+        // Chcek for other overlapping Period
+        $qb = $this->findCollectionQueryBuilder(Period::class, 'p', null)
+            ->where('p.camp = :camp')
+            ->andWhere('p.id != :id')
+            ->andWhere('p.start < :end')
+            ->andWhere('p.end > :start')
```

Comment:
Müsste es hier nicht <= und >= heissen? Sonst wären Perioden die sich nur in einem Tag überlappen erlaubt, sehe ich das falsch?

Reply:
Stimmt, passe ich an